chainerx_cc/chainerx/routines/activation.cc

```diff
@@ -91,6 +91,34 @@ Array LeakyRelu(const Array& x, Scalar slope) {
     return Where(x_cast >= zero, x_cast, slope * x_cast);
 }
 
+std::vector<Array> TreeLstm(std::vector<Array> arrays) {
+    size_t n_ary = arrays.size() - 1;
+    std::vector<Array> gates = ExtractGates(arrays[arrays.size() - 1], 3 + n_ary, 1);
+
+    Array a = Squeeze(gates[0]);
+    Array i = Squeeze(gates[1]);
+    Array o = Squeeze(gates[2]);
+    std::vector<Array> fs;
+    for (size_t i = 3; i < gates.size(); i++) {
+        fs.emplace_back(Squeeze(gates[i]));
+    }
+    Array a_ = Tanh(a);
+    Array i_ = Sigmoid(i);
+    Array o_ = Sigmoid(o);
+    std::vector<Array> fs_s{};
+    for (const auto & f : fs) {
```

Comment:
I think this space should be removed (`auto &`)

Reply:
Done